libcxx/docs/ReleaseNotes/22.rst

```diff
@@ -41,6 +41,8 @@ Implemented Papers
 - P2321R2: ``zip`` (`Github <https://github.com/llvm/llvm-project/issues/105169>`__) (The paper is partially
   implemented. ``zip_transform_view`` is implemented in this release)
 - P3168R2: Give ``std::optional`` Range Support (`Github <https://github.com/llvm/llvm-project/issues/105430>`__)
+- P0881R7: A Proposal to add stacktrace library (`Github <https://github.com/llvm/llvm-project/issues/105131>`__)
+- P2301R1: Add a `pmr` alias for `std::stacktrace` (`Github <https://github.com/llvm/llvm-project/issues/105167>`__)
```

Comment:
```suggestion
- P0881R7: A Proposal to add stacktrace library (`Github <https://llvm.org/PR105131>`__)
- P2301R1: Add a `pmr` alias for `std::stacktrace` (`Github <https://llvm.org/PR105167>`__)
```

Please update all issue links to the universal format after: https://github.com/llvm/llvm-project/pull/156288